Makefile

```diff
@@ -24,5 +24,9 @@ nose3: qless-core
 	rm -rf .coverage
 	nosetests3 --exe --cover-package=qless --with-coverage --cover-branches -v
 
+.PHONY: requirements
+requirements:
```

Comment:
Actually, we don't need the `.PHONY` at all. But if we make the target `requirements.txt`, it says that it's up-to-date. It makes sense that it would -- we list no dependencies, and the file exists.